Penteract/Assets/Scripts/AIMeleeGrunt.cpp

```diff
@@ -405,6 +423,7 @@ void AIMeleeGrunt::OnCollision(GameObject& collidedWith, float3 collisionNormal,
 					if (animation->GetCurrentState()) {
 						animation->SendTrigger(animation->GetCurrentState()->name + "StunStart");
 					}
+					particlesEmp->PlayChildParticles();
```

Comment:
Check nullity. In the start you check the nullity of the aux local variable, but at this point this pointer could be null